README.rst

```diff
@@ -0,0 +1,119 @@
+Firefox Accounts Page Object Model
+==================================
+
+`Selenium WebDriver <http://docs.seleniumhq.org/docs/03_webdriver.jsp>`_ compatible page object model and utilities for `Firefox Accounts <https://accounts.firefox.com>`_
+
+.. image:: https://img.shields.io/pypi/l/fxapom.svg
+   :target: https://github.com/mozilla/fxapom/blob/master/LICENSE
```

Comment:
Oops, this links to a file that doesn't exist. Let's create a LICENSE file and remove that section from the README.